ansible/deploy_slave.yml

```diff
@@ -4,25 +4,18 @@
   vars:
     ansible_ssh_pipelining: yes
   tasks:
-    - name: Add contrib to sources.list
-      blockinfile:
-        path: /etc/apt/sources.list
-        block: |
-          deb http://192.168.122.1:3142/ftp.de.debian.org/debian/ stretch main contrib
-          deb-src http://192.168.122.1:3142/ftp.de.debian.org/debian/ stretch main contrib
```

Comment:
Nice, we finally get rid of needing contrib?

Reply:
I think only `flashplugin-nonfree` was in contrib and as it was a dependency of OpenWPM, which was replaced by privacyscanner, it is gone now I guess :)